src/flowops_library.c

```diff
@@ -22,7 +22,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-#include "config.h"
+#include "../config.h"
```

Comment:
Before I made this change I looked through uperf code and checked how config is included. In most places it is included via path specification so I changed here for consistency.